bitnami/dataplatform-bp1/values.yaml

```diff
@@ -131,12 +178,32 @@ spark:
       requests:
         cpu: 250m
         memory: 5120Mi
+  ## Metrics configuration
+  ##
+  metrics:
+    enabled: false
+    ## Annotations for the Prometheus metrics on master nodes
+    ##
+    masterAnnotations:
+      prometheus.io/scrape: 'true'
+      prometheus.io/path: '/metrics/'
+      prometheus.io/port: '{{ .Values.master.webPort }}'
+    ## Annotations for the Prometheus metrics on worker nodes
+    ##
+    workerAnnotations:
+      prometheus.io/scrape: 'true'
+      prometheus.io/path: '/metrics/'
+      prometheus.io/port: '{{ .Values.worker.webPort }}'
```

Comment:
Same here

Reply:
Thanks for pointing this out. I updated the port to `{{ .values.spark.worker.webPort}}`